### ISSUE TITLE
Access to validation loss curve

### ISSUE THREAD

**Issue by ArthPelt5** [Nixtla/neuralforecast#1333]
### Description

I cannot find in the documentation how to extract the values of the validation losse during steps. I want to use it to plot the learning curves of my models to check their performance.

For my `TSMixerx` model, I can only access to `train_loss_step` to have the training curve. Here is the related code : 
```
model_tsmixerx = TSMixerx(
    h=horizon, # horizon = 30
    n_series=n_series, # n_series = 6
    input_size=input_size, # input_size = 60
    futr_exog_list=features, # List of exogenous features
    valid_loss=MAE(),
    val_check_steps=100,
    n_block=2,
    learning_rate=0.001,
    ff_dim=32,
    max_steps=2000,
    batch_size=128,
    dropout=0.5,
    random_seed=1,
    step_size=1
)
```
Here is the last line of the training part : 
```
Epoch 1999: 100%|██████████| 1/1 [00:00<00:00, 58.93it/s, v_num=27, train_loss_step=0.223, train_loss_epoch=0.223]
```

Why there is not information about validation (`valid_loss_step`) ?

**Comment by marcopeix** [comment]
When you fit, did you specify a validation size? Something like `nf.fit(df, val_size=100)` for example. That way, 100 time steps are reserved for the validation set, and so a validation loss is calculated.

**Comment by ArthPelt5** [comment]
Ok I see. After I just have to create a callback object to extract the informations ?

**Comment by marcopeix** [comment]
Or you can do something like this:
```python
mdl = nf.models[0]
train = mdl.train_trajectories  
valid = mdl.valid_trajectories

df_loss = pd.DataFrame(train, columns=['x', 'y'])
plt.plot(df_loss['x'], df_loss['y'])
plt.title("Training Loss") 
plt.show()

df_loss = pd.DataFrame(valid, columns=['x', 'y'])
plt.plot(df_loss['x'], df_loss['y']) 
plt.title("Validation Loss")
plt.show()
```
